src/app/zap-templates/zcl/data-model/chip/dishwasher-control-cluster.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2023 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="Appliances"/>
+
+  <bitmap name="DishwasherControlFeature" type="BITMAP32">
+    <cluster code="0x0058" />
+    <field name="STEAMWASH" mask="0x01" />
+    <field name="HEATEDDRY" mask="0x02" />
+    <field name="EXTDRY" mask="0x04" />
```

Comment:
The field names should be the feature names, not the feature codes, to match other clusters.  So:
```suggestion
    <field name="SteamWash" mask="0x01" />
    <field name="HeatedDry" mask="0x02" />
    <field name="ExtendedDry" mask="0x04" />
```